source/frontend/src/features/mapSideBar/property/tabs/takes/detail/TakesDetailView.tsx

```diff
@@ -182,14 +182,10 @@ export const TakesDetailView: React.FunctionComponent<ITakesDetailViewProps> = (
                 <StyledBorderSection>
                   <SectionField
                     label="Is there a new highway dedication? *"
-                    labelWidth="8"
+                    labelWidth="9"
                     tooltip="The term new highway dedication includes municipal road or provincial public highway."
                   >
-                    <YesNoButtons
-                      id="newRightOfWayToggle"
-                      disabled
-                      value={take.isNewHighwayDedication ?? undefined}
-                    />
+                    {booleanToYesNoUnknownString(take.isNewHighwayDedication ?? undefined)}
```

Comment:
nit: Probably you don't need the '??' operator, since the property is 'bool | null' and the function accepts 'bool | null'.

Reply:
same for others in this component.